### PR TITLE
Added design guidelines for filter and sort

### PR DESCRIPTION
**Pull request by Blackbaud-DeniseCisneros**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-lists?_sp.spa=pr-design-guidelines-filter-sort](https://developer.blackbaud.com/skyux-lists?_sp.spa=pr-design-guidelines-filter-sort)

File: package.json [hunk 1]
```diff
@@ -33,7 +33,8 @@
     "@skyux/popovers": "^4.0.0"
   },
   "dependencies": {
-    "dragula": "3.7.2",
+    "@blackbaud/skyux-builder-plugin-code-block": "1.1.0",
```
Comment (Blackbaud-AlexKingman) [inline package.json:36]: ☝️ This should be moved to a dev-dependency.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-lists?_sp.spa=pr-design-guidelines-filter-sort](https://developer.blackbaud.com/skyux-lists?_sp.spa=pr-design-guidelines-filter-sort)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-lists?_sp.spa=pr-design-guidelines-filter-sort](https://developer.blackbaud.com/skyux-lists?_sp.spa=pr-design-guidelines-filter-sort)

**Review by blackbaud-johnly — APPROVED** [review]

**Pull request merged**